client-app/ui-kit/composables/useQuantityValidationSchema.test.ts

```diff
@@ -0,0 +1,121 @@
+import { describe, expect, it } from "vitest";
```

Comment:
One of important things about our validation is what it not only tells if quantity is invalid, but also choose one of 4 predefined messages. These tests doesn't check which message will be displayed.